src/ContaoCommunityAlliance/DcGeneral/Contao/View/Contao2BackendView/EditMask.php

```diff
@@ -713,4 +716,63 @@ class_implements(
 
         return $objTemplate->parse();
     }
+
+    /**
+     * Get the breadcrumb navigation via event.
+     *
+     * @return string
+     *
+     * @SuppressWarnings(PHPMD.Superglobals)
+     * @SuppressWarnings(PHPMD.CamelCaseVariableName)
+     */
+    protected function breadcrumb()
+    {
+        $event = new GetBreadcrumbEvent($this->getEnvironment());
+
+        $this->getEnvironment()->getEventDispatcher()->dispatch(sprintf('%s', $event::NAME), $event);
+
+        $arrReturn = $event->getElements();
```

Comment:
Please do not use type prefixes in variables in new code. We are changing the codebase to not use prefixes step by step. Therefore we should not introduce them with new code.